src/components/Countdown.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react'
+
+function Countdown({ Date }) {
+  const [date, setDate] = useState(Date)
+
+  console.log(date)
+    debugger
+
+  return (
+    <div className="grid grid-cols-4 flex-row flex-wrap items-end justify-around px-10 text-center uppercase">
+      <div className="after:absolute after:text-2xl after:content-[':']">
+        <div>days</div>
+        <div className="text-2xl">{date.days()}</div>
```

Comment:
why not to execute function once, and then use all values: days, hours, mInutes, seconds.
Instead of executing function each time in several component places, retrieve function values and use them directly.